docs/manage-table-partition-using-gravitino.md

```diff
@@ -19,13 +19,13 @@ Although many catalogs inherently manage partitions automatically, there are sce
 
 The following table shows the partition operations supported across various catalogs in Gravitino:
 
-| Operation             | Hive catalog                                                       | Iceberg catalog                                                    | Jdbc-Mysql catalog | Jdbc-PostgreSQL catalog |
-|-----------------------|--------------------------------------------------------------------|--------------------------------------------------------------------|--------------------|-------------------------|
-| Add Partition         | YES                                                                | NO                                                                 | NO                 | NO                      |
-| Get Partition by Name | YES                                                                | NO                                                                 | NO                 | NO                      |
-| List Partition Names  | YES                                                                | NO                                                                 | NO                 | NO                      |
-| List Partitions       | YES                                                                | NO                                                                 | NO                 | NO                      |
-| Drop Partition        | [Coming Soon](https://github.com/datastrato/gravitino/issues/1655) | [Coming Soon](https://github.com/datastrato/gravitino/issues/1655) | NO                 | NO                      |
+| Operation             | Hive catalog                                                                  | Iceberg catalog                                                               | Jdbc-Mysql catalog | Jdbc-PostgreSQL catalog |
+|-----------------------|-------------------------------------------------------------------------------|-------------------------------------------------------------------------------|--------------------|-------------------------|
+| Add Partition         | &#10004;                                                                      | &#10008;                                                                      | &#10008;           | &#10008;                |
```

Comment:
It may be confusing if users only read from markdown document instead of website.

Reply:
Markdown also can render the HTML entity:

<img width="801" alt="image" src="https://github.com/datastrato/gravitino/assets/24897598/49b6e765-fdb1-451e-aa74-be7231959f82">

Reply:
Cool!